src/patternfly/components/Badge/Badge.hbs

```diff
@@ -0,0 +1,3 @@
+<span class="pf-c-badge {{ badgeClass }}">
```

Comment:
There was conversation about the format changing to `pf-c-badge--modifierClass` which I did in alerts but we need to decide moving forward.

Reply:
Also, I think the convention was kebab case for handlebars names, so this file should be `badge.hbs` and wherever badge is brought in, it should be referenced by `{{#> badge }}`

Reply:
Ah sorry, I copied the style used in Buttons. I'll update this.